gitconfig

```diff
@@ -11,16 +11,8 @@
   ca = commit --amend -v
   ci = commit -v
   co = checkout
-  co-pr = !sh -c 'git fetch origin pull/$1/head:pr/$1 && git checkout pr/$1' -
-  create-branch = !sh -c 'git push origin HEAD:refs/heads/$1 && git fetch origin && git branch --track $1 origin/$1 && cd . && git checkout $1' -
   ctags = "!sh -c '[ -f .git/hooks/ctags ] || git init; .git/hooks/ctags' git-ctags"
-  current-branch = !sh -c 'git rev-parse --abbrev-ref HEAD' -
-  delete-branch = !sh -c 'git push origin :refs/heads/$1 && git branch -D $1' -
-  merge-branch = !git checkout master && git merge @{-1}
-  pr = !hub pull-request
```

Comment:
It's [not aliased](https://github.com/thoughtbot/dotfiles/blob/master/aliases). I personally prefer to maintain the distinction between `git` and `hub`.